pkg/localmetrics/localmetrics.go

```diff
@@ -0,0 +1,49 @@
+package localmetrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	metricsTag = "ocmagentoperator"
+	nameLabel  = "ocmagent_name"
+)
+
+var (
+	MetricPullSecretInvalid = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Subsystem: metricsTag,
+		Name:      "ocm_agent_pull_secret_invalid",
+		Help:      "Failed to obtain a valid pull secret",
+	}, []string{nameLabel})
+
+	MetricOcmAgentResourceAbsent = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Subsystem: metricsTag,
+		Name:      "ocm_agent_resource_absent",
+		Help:      "No OCM Agent resource found",
+	}, []string{})
```

Comment:
Maybe we can have a label here to reflect the resource type?
But I doubt if the condition here would happen anyway.

Reply:
I figured the resource type is kind of implied by the metric name of `ocm_agent_resource`. 
But yeah.. I am in agreement that this metric should be almost always 0.